File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as chromeService
from webdriver_manager.chrome import ChromeDriverManager
from selenium.webdriver.common.by import By
service=chromeService(ChromeDriverManager().install())
with webdriver.Chrome(service=service) as driver:
    driver.get("https://v1.training-support.net")
    print("Webpage title : ",driver.title)
    driver.find_element(By.ID,"about-link").click()
    print("2nd Webpage title : ",driver.title)
    driver.quit()

File: Selenium/Activities/Activity1.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[]args){
        WebDriverManager.chromedriver().setup();
        WebDriver driver=new ChromeDriver();
        driver.get("https://v1.training-support.net");
        System.out.println("Title of WebPage : "+driver.getTitle());
        driver.findElement(By.id("about-link")).click();
        System.out.println("Title of 2nd WebPage : "+driver.getTitle());
        driver.quit();
    }
}
